### ISSUE TITLE
[Suggestion] Pause highway builder on auto eat

### ISSUE THREAD

**Issue by RedCarlos26** [MeteorDevelopment/meteor-client#2179]
### Describe the feature

Add an option into the auto-gap and auto eat module so it pauses highway builder for the time it eats and then reenables it.

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client.
(I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested.
(I have checked every suggestion on github)

- [X] This feature is a valid suggestion.
(I have read the FAQs and Guides)

**Comment by xXBaboBastiXx** [comment]
Hey, I have also issues with auto eat in meteor builds on version 1.18.2, including the newest build 1452.

In my case, auto eat is able to pause highway builder, but the player keeps stuck in the eating food animation. While stucking in the eating animation, highway builder continues to move foreward, but stops to break blocks, also the speed of the player decreases hard.
Disabling and re-enabling auto eat-doesn't help, auto eat makes the player getting stuck in the eat animation again after re-enabling auto eat. 
This happens only in newer builds, while auto eat works completely fine in any 1.18.1 build and below.

In other words: your suggestion exists already in older builds, but doesn't work in the newest version of meteor.

**Comment by RedCarlos26** [comment]
It doesn't work in older versions as well.
